### PR TITLE
add remove dummies in dcm2nii pipeline

### PR DESCRIPTION
**Pull request by mohmdrezk**
A python script is called and removed the number of requested dummies
Dependancies: Nibabel should be installed

### CODE REVIEW

File: removeDummies.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/python
```
Comment (mohmdrezk) [inline removeDummies.py]: This is the python script that will remove dummies. I would suggest that someone else checks it with a nifti image to double check that everything is as expected

Comment (mohmdrezk) [reply removeDummies.py]: @Remi-Gau & @marcobarilari : would you be interested in testing the script (no need to run the pipeline, just the script alone with any nifti image you have)

File: removeDummies.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        print(header)
+
+    # Get the affine matrix
+    affine = img.affine
+    image_data = img.get_fdata()    # Get the 4D matrix with the voxel values
+    dimOriginal = image_data.shape  # Get the shape of the matrix (Original 4D dimensions)
+
+    # Remove the requested dummies in the 4th dimensions
+    image_data = (image_data[:,:,:,numDummies:])
+    image_data = image_data.astype(int)      # Change the type of numbers to int
```
Comment (mohmdrezk) [inline removeDummies.py]: for an unknown reason (to me) when the nifti images were read the numbers were who were originally integers and converted to float, and that lead to the output image to have different numbers (usually decimals) from the original  image. 
In this step, I am changing the type of the numbers to int before saving and that seems to solve the issue. 
When you try the script, please as a double check, see if the numbers are identical and the process didnt alter the voxel values.

File: removeDummies.py (hunk 1 above)
Comment (mohmdrezk) [reply removeDummies.py]: the voxel values are always "int" in the converted dicoms , right?

Comment (Remi-Gau) [reply removeDummies.py]: > @Remi-Gau & @marcobarilari : would you be interested in testing the script (no need to run the pipeline, just the script alone with any nifti image you have)

Will have a look.

Comment (Remi-Gau) [reply removeDummies.py]: ok finally took the time to check this. it works for me. 

Sent a PR to your branch to fix some style guide issues to abide by the pep8 standards

File: removeDummies.py (hunk 2 above)
Comment (Remi-Gau) [reply removeDummies.py]: I checked that too and it is fine on my side

**Comment by marcobarilari** [comment]
Hey, I'll check it asap (if still need reviewing). I'm sorry for the delay.

**Comment by mohmdrezk** [comment]
no worries, its ok now.

**Pull request merged**